deploifai/utilities/config/dataset_config.py

```diff
@@ -0,0 +1,108 @@
+import configparser
+import pathlib
+import click
+
+from deploifai.utilities.config.find_config_filepath import find_config_absolute_path
+
+"""
+Manages a .dataset.cfg file to store configuration info about a project.
+
+config dictionary structure:
+{
+    "DATASET": {"id": string}
+}
+"""
+
+relative_config_filepath = pathlib.Path("dataset.cfg")
+
+config_file_path = find_config_absolute_path(relative_config_filepath)
+
+config_sections = ["DATASET"]
+
+
+class DeploifaiDataAlreadyInitialisedError(Exception):
+    """
+    Exception when a data directory has been initialised through Deploifai in an ML project.
+    """
+
+    def __init__(self, message):
+        super(DeploifaiDataAlreadyInitialisedError, self).__init__(message)
+
+
+def create_config_files():
+    """
+    Creates the .dataset config files.
+    :return: None
+    """
+    global config_file_path
+    dataset_config_dir = pathlib.Path()
+    dataset_config_dir.mkdir(exist_ok=True)
```

Comment:
This actually doesn't seem right to me. The dataset config directory should be the current working directory. Seems like `pathlib.Path()` would do that, but it doesn't make sense to do a `mkdir` here.

Reply:
yes
i was also slightly confused about this before
but this was for making .deploifai directory in local config
should have removed it before